docs/Demo.md

```diff
@@ -1,447 +1,409 @@
 # Pelorus Demo
 
 ### Assumptions
-- oc, helm command line tools installed
+- OpenShift Container, Helm command line tools installed
```

Comment:
I would explicitly say what are the CLI binaries required in one or other form so maybe:
- OpenShift Containr (oc), Helm (helm) command line tools installed?